indico/web/client/js/jquery/widgets/jinja/category_picker_widget.js

```diff
@@ -51,6 +52,8 @@
         openInDialog: true,
         actionOn,
         onAction(category) {
+          $categoryWarning.toggleClass('hidden', !category.has_children || category.has_events);
```

Comment:
We also need to toggle it for the initially-prefilled category - right now we don't show it there, even though that's probably the most important time to show it.

PS: Maybe move this one after the variable definitions? ;)